boards/adi/max32670evkit/doc/index.rst

```diff
@@ -118,6 +118,8 @@ Below interfaces are supported by Zephyr on MAX32670EVKIT.
 +-----------+------------+-------------------------------------+
 | Timer     | on-chip    | counter                             |
 +-----------+------------+-------------------------------------+
+| PWM       | on-chip    | pwm                                 |
```

Comment:
Could this line be consistent, you should use one of them either this

| PWM       | on-chip    | pwm 
or
| Timer     | on-chip    | counter, pwm

Reply:
Done